components/Dialogs/src/MDCAlertControllerView.h

```diff
@@ -26,6 +26,7 @@
 
 @property(nonatomic, strong, nullable) UIFont *buttonFont UI_APPEARANCE_SELECTOR;
 @property(nonatomic, strong, nullable) UIColor *buttonColor UI_APPEARANCE_SELECTOR;
+@property(nonatomic, strong, nullable) UIColor *buttonInkColor UI_APPEARANCE_SELECTOR;
```

Comment:
Do we need to add the UI_APPEARANCE_SELECTOR, I thought the plan was to deprecate that?

Reply:
This class still supports it, so I think it makes sense to continue supporting it until it's either too challenging to support or we explicitly remove it.